opentelemetry-sdk/src/opentelemetry/sdk/_metrics/export/__init__.py

```diff
@@ -11,3 +11,71 @@
 # WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
 # See the License for the specific language governing permissions and
 # limitations under the License.
+
+from abc import ABC, abstractmethod
+from enum import Enum
+from os import linesep
+from sys import stdout
+from typing import IO, Callable, Sequence
+
+from opentelemetry.sdk._metrics.data import MetricData
+from opentelemetry.sdk._metrics.measurement import Measurement
+
+
+class MetricExportResult(Enum):
+    SUCCESS = 0
+    FAILURE = 1
+
+
+class MetricExporter(ABC):
+    """Interface for exporting metrics.
+
+    Interface to be implemented by services that want to export metrics received
+    in their own format.
+    """
+
+    def export(self, metrics: Sequence[MetricData]) -> "MetricExportResult":
+        """Exports a batch of telemetry data.
+
+        Args:
+            metrics: The list of `opentelemetry.trace.Span` objects to be exported
+
+        Returns:
+            The result of the export
+        """
+
+    @abstractmethod
+    def shutdown(self) -> None:
+        """Shuts down the exporter.
+
+        Called when the SDK is shut down.
+        """
+
+
+class ConsoleMetricExporter(MetricExporter):
+    """Implementation of :class:`MetricExporter` that prints metrics to the
+    console.
+
+    This class can be used for diagnostic purposes. It prints the exported
+    metrics to the console STDOUT.
+    """
+
+    def __init__(
+        self,
+        out: IO = stdout,
+        formatter: Callable[
+            [Measurement], str
```

Comment:
Should this be `Metric` instead of `Measurement`?

Reply:
Or they could possibly represent the same thing?

Reply:
Metric is correct here.